### PR TITLE
Tips mobile view fix

### PR DESCRIPTION
**Pull request by wigsten**

### CODE REVIEW

File: pushToS3.ps1 [hunk 1]
```diff
@@ -32,7 +32,7 @@ foreach ($item in $items)
 			$folder = $folder -replace "\\", "/";
 			$folder = $folder -replace $outputFolderEscaped, "";
 			#$folder = $folder -replace "/", "\";
-
+
```
[the remark this answers is not in the document]
Comment (wigsten) [reply pushToS3.ps1]: Are you talking about pushToS3.ps1?

File: src/js/app/components/boxed-window.js [hunk 2]
```diff
@@ -14,26 +14,45 @@
...
-      this.toggleMoreInfoIcon(el);
+      this.toggleMoreInfoIcon(el, overwrite);
     }
   }
 
-  BoxedWindow.prototype.toggleMoreInfoIcon = function(el) {
-    if ( ! this.expanded ) {
+  BoxedWindow.prototype.toggleMoreInfoIcon = function(el, overwrite) {
+    // if toggle all accordions; eg. tips page
+    if (overwrite !== undefined) {
```
[the remark this answers is not in the document]
Comment (wigsten) [reply src/js/app/components/boxed-window.js]: I agree, it's not ideal - however the scope is a tad problematic. I'll see if I can improve

File: pushToS3.ps1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (singuyen) [reply pushToS3.ps1]: Yes.

[the remark this answers is not in the document]
Comment (singuyen) [reply pushToS3.ps1]: Just noticed it includes the changes which is already on dev. Can you rebase it?

[the remark this answers is not in the document]
Comment (wigsten) [reply pushToS3.ps1]: Strange... I always rebase when setting up for PR. Works now thou except for the trailing white space (also, this isnt a VR file, seems to be a S3 config file of some kind)

File: src/js/app/components/boxed-window.js [hunk 3]
```diff
@@ -14,26 +14,35 @@
...
     if ( el.querySelector('.i-moreinfo') !== null || el.querySelector('.i-lessinfo') !== null ) {
-      this.toggleMoreInfoIcon(el);
+      this.toggleMoreInfoIcon(el, overwrite);
     }
   }
 
-  BoxedWindow.prototype.toggleMoreInfoIcon = function(el) {
-    if ( ! this.expanded ) {
+  BoxedWindow.prototype.toggleMoreInfoIcon = function(el, overwrite) {
+    var isOpen = $(el).find('.i-moreinfo').length;
```
Comment (wigsten) [inline src/js/app/components/boxed-window.js]: Hey Si, good call on this one. I had a problem with Scope earlier and left this by mistake. Also your comment disappeared for some reason - but here's the update

**Pull request merged**